Jenkinsfile

```diff
@@ -161,8 +161,9 @@ pipeline {
                         // unstash "buildLogs"
                         // TODO correct flags for rsync
                         sh "ssh ${LOG_SERVER_USER}@${LOG_SERVER} \"mkdir -p ${LOG_ROOT_DIR}/${BUILD_SPECIFIC_DIR}\""
-                        obtainLogFile env.JOB_NAME, env.BUILD_ID, "clean_log.txt.gz"
-                        sh "rsync clean_log.txt.gz ${LOG_SERVER_USER}@${LOG_SERVER}:${REMOTE_DST_FILE}"
+                        def tmpLogFilename = "clean_log.txt.gz"
+                        obtainLogFile env.JOB_NAME, env.BUILD_ID, tmpLogFilename
```

Comment:
I see ```()```  everywhere a groovy script is called and I think we should choose one option, we can discuss with other members of our team and decide which convention to use.